plugins/generator-1.14.4/forge-1.14.4/templates/json/plant_double_bottom.json.ftl

```diff
@@ -2,6 +2,7 @@
     "parent": "block/cross",
     "textures": {
         "cross": "${modid}:blocks/${data.textureBottom?has_content?then(data.textureBottom, data.texture)}",
-        "particle": "${modid}:blocks/${data.textureBottom?has_content?then(data.textureBottom, data.texture)}"
+        "particle": "${modid}:blocks/<#if data.particleTexture?has_content>${data.particleTexture}<#else>

```

Comment:
Lines like these will need to be changed. Code generated for these looks like this:

![image](https://user-images.githubusercontent.com/16374228/96119517-63d6ed80-0eed-11eb-9981-c466191719da.png)

You will need to keep all of this in single line, you need to respect JSON rules in templates as things like this can't be reformatted later. String can't have line breaks, for example. Verify all JSON files changed for this and test in MC if all works, I noticed this when testing double plant ;)

The same applies for 1.15.2.

Reply:
Should be fixed now, the only template with a line break inside the if statement was the double plant bottom half one